apps/music/js/ui/views/search_view.js

```diff
@@ -90,18 +85,15 @@ var SearchView = {
         this.searchHandles[option] = null;
         return;
       }
-      var resultLowerCased = result.metadata[option].toLocaleLowerCase();
-      if (Normalizer.toAscii(resultLowerCased).indexOf(query) !== -1) {
-        this.dataSource.push(result);
-
-        numResults[option]++;
-        lists[option].classList.remove('hidden');
-        lists[option].getElementsByClassName('search-result-count')[0]
-                     .textContent = numResults[option];
-        lists[option].getElementsByClassName('search-results')[0].appendChild(
-          createListElement(option, result, this.dataSource.length - 1, query)
-        );
-      }
+
+      this.dataSource.push(result);
+      numResults[option]++;
+      lists[option].classList.remove('hidden');
+      lists[option].getElementsByClassName('search-result-count')[0]
```

Comment:
Probably not worth improving upon this view code much at this point, but this could be expressed a bit cleaner via:

```
lists[option].querySelector('.search-result-count').textContent = numResults[option];
```